src/Symfony/Component/Routing/Matcher/Dumper/CompiledUrlMatcherDumper.php

```diff
@@ -121,7 +121,7 @@ static function (\$condition, \$context, \$request) { // \$checkCondition
         }
     }
 EOF;
-            $compiledRoutes[4] = $forDump ? $checkConditionCode .= ",\n" : eval('return '.$checkConditionCode.';');
+            $compiledRoutes[4] = $forDump ? $checkConditionCode.",\n" : eval('return '.$checkConditionCode.';');
```

Comment:
I've seen this updated in the previous PR also and still merged - but FYI this is not exactly the same: using `.=` provides better memory usage (maybe the engine does the optimization already, I don't know).
But ok to change anyway.

Reply:
Thanks for the explanation. I assumed a simple concat is faster than an assign + concat. It might worth it to take a look or to revert those changes. I don't want to cause a performance regression just by trying to "clean" the code.